installing/installing_vsphere/installation-config-parameters-vsphere.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: ASSEMBLY
+[id="installation-config-parameters-vsphere"]
+= Installation configuration parameters
+include::_attributes/common-attributes.adoc[]
+:context: installation-config-parameters-vsphere
+
+toc::[]
+
+Before you deploy an {product-title} cluster, you provide parameter values to describe your account on the cloud platform that hosts your cluster and optionally customize your cluster's platform. When you create the `install-config.yaml` installation configuration file, you provide values for the required parameters through the command line. If you customize your cluster, you can modify the `install-config.yaml` file to provide more details about the platform.
```

Comment:
@kalexand-rh That is because it is not just vSphere installation parameters. :smile: 

The module is comprised of required and optional parameters that are platform agnostic, plus platform specific parameters. In this case, vSphere.  Thanks for pointing this out, however, as I think this refactor should point out that the tables represent required, optional, a platform specific parameters.

Reply:
@kalexand-rh Oh I see what you mean now. I will update the abstract to specify that these are installation parameters used to install a cluster on vSphere.